docs/static/releasenotes.asciidoc

```diff
@@ -234,122 +130,30 @@ for content in XML elements. This option allows you to force text content and at
 a hash value (https://github.com/logstash-plugins/logstash-filter-xml/issues/14[Issue 14]).
 * Fixed a bug that ensures that a `target` is set when storing XML content in the event (`store_xml => true`).
 
-[[alpha2]]
-=== Logstash 5.0-alpha2 Release Notes
-
-* Added the `--preserve` option to the `bin/logstash-plugin` install command. This option allows you to preserve gem options that are already specified in the `Gemfile`. Previously, these options were overwritten.
-* Added support for `DEBUG=1` when running any plugin-related commands. This option gives you a bit more information about what the bundler is doing.
-* Added reload support to the init script so you can do `service logstash reload`.
-* Fixed use of the `KILL_ON_STOP_TIMEOUT` variable in the init script to allow Logstash to force stop ({lsissue}4991[Issue 4991]).
-* Upgraded to JRuby 1.7.25.
-* Renamed filenames for Debian and RPM artifacts to match Elasticsearch's naming scheme. The metadata is still the same, so upgrades will not be affected. If you have automated downloads for Logstash, please make sure you use the updated URLs ({lsissue}5100[Issue 5100]). 
-
-[float]
-==== Input Plugins
-
-*`Kafka`*:
-
-* Fixed an issue where Snappy and LZ4 compression were not working.
-
-[float]
-==== Filter Plugins
-
-*`GeoIP`*:
-
-* Added support for the GeoIP2 city database and support for IPv6 lookups (https://github.com/logstash-plugins/logstash-filter-geoip/issues/23[Issue 23]).
-
 [float]
 ==== Output Plugins
 
 *`Elasticsearch`*:
 
+* Breaking Change: The index template for 5.0 has been changed to reflect Elasticsearch's mapping changes. Most importantly,
+the subfield for string multi-fields has changed from `.raw` to `.keyword` to match Elasticsearch's default behavior
+(https://github.com/logstash-plugins/logstash-output-elasticsearch/issues/386[Issue 386]). See <<breaking-changes>> for details about how this change affects new and existing users.
+* Added `check_connection_timeout` parameter, which has a default of 10m.
+* Added the ability for the plugin to choose which default template to use based on the Elasticsearch version (https://github.com/logstash-plugins/logstash-output-elasticsearch/issues/401[Issue 401]).
+* Elasticserach output is now fully threadsafe. This means internal resources can be shared among multiple
+`output { elasticsearch {} }` instances.
+* Added sniffing improvements so any current connections don't have to be closed/reopened after a sniff round.
+* Introduced a connection pool to efficiently reuse connections to Elasticsearch backends.
```

Comment:
I think we can do without the 'efficiently' word. pooling and connection reuse *are* efficiency improvements :)